share/completions/s3cmd.fish

```diff
@@ -0,0 +1,121 @@
+# s3cmd
+
+function __s3cmd_is_remote_path
+	commandline -ct | string match -r -- "^s3://.*"
+end
+
+function __s3cmd_is_valid_remote_path
+	commandline -ct | string match -r -- "^s3://.*?/.*"
```

Comment:
`.*?`? The `?` is redundant - `*` already allows matching nothing.

(Also the ending `.*` is unnecessary, since regex-matching allows partial matches anyway)